js/components/controlBar.js

```diff
@@ -310,7 +306,7 @@ var ControlBar = React.createClass({
       fullscreenIcon = "expand";
       fullscreenAriaLabel = CONSTANTS.ARIA_LABELS.FULLSCREEN;
     }
-
+    
```

Comment:
let's remove extra whitespaces on all the files/

Reply:
fix